_sass/kroxylicious.scss

```diff
@@ -49,7 +41,7 @@ $light: $kroxy-light;
   padding: 0.75rem 0;
   background-color: transparent;
   background: linear-gradient(to bottom, rgba($kroxy-dark, 1), rgba($kroxy-dark, 0.95));
-  box-shadow: 0 0.5rem 1rem $kroxy-shadow-dark,inset 0 -1px 0 $kroxy-shadow-dark;
+  box-shadow: 0 0.5rem 1rem rgba($kroxy-dark, 0.15),inset 0 -1px 0 rgba($kroxy-light, 0.15);
```

Comment:
Is there a reason you've moved from pre computing `rgba($kroxy-dark, 0.15)` as `$kroxy-shadow-dark` to calling it inline?

Reply:
This variable is just a transparency of `$kroxy-dark`, and it's only used in one place. We compute other transparencies of this variable elsewhere already (for example the line above this one), so given this scss file is compiled at build time by jekyll-sass-converter into regular CSS anyway, it felt a little redundant to keep it around.